shopping_mall/backend/tests/conftest.py

```
@@ -0,0 +1,242 @@
+    user_id=10,
+    total_price=35000,
+    status="shipping",
+    created_at=None,
+    items=None,
+):
+    order = MagicMock()
+    order.id = order_id
+    order.user_id = user_id
+    order.total_price = total_price
+    order.status = status
+    order.created_at = created_at or datetime(2026, 4, 10)
```

Ruff (0.15.10)

[warning] `datetime.datetime()` called without a `tzinfo` argument

(DTZ001)